docs/cue/reference/components/sources/host_metrics.cue

```diff
@@ -376,6 +376,7 @@ components: sources: host_metrics: {
 		filesystem_free_bytes:  _host & _filesystem_bytes & {description: "The number of bytes free on the named filesystem."}
 		filesystem_total_bytes: _host & _filesystem_bytes & {description: "The total number of bytes in the named filesystem."}
 		filesystem_used_bytes:  _host & _filesystem_bytes & {description: "The number of bytes used on the named filesystem."}
+		filesystem_used_ratio:  _host & _filesystem_bytes & {description: "The ratio between used and free on the named filesystem."}
```

Comment:
I think this would actually be the ratio of used space as a fraction of total space, not free space. I know the `heim` crate says what you have here, but that kind of ratio doesn't make much sense.

Reply:
Good point, though example from my laptop:

```json
{"name":"filesystem_free_bytes","namespace":"host","tags":{"collector":"filesystem","device":"/dev/disk1s2","filesystem":"apfs","host":"COMP-C02F514NML87","mountpoint":"/System/Volumes/Data"},"timestamp":"2021-07-20T00:55:59.015603Z","kind":"absolute","gauge":{"value":281301454848.0}}
{"name":"filesystem_total_bytes","namespace":"host","tags":{"collector":"filesystem","device":"/dev/disk1s2","filesystem":"apfs","host":"COMP-C02F514NML87","mountpoint":"/System/Volumes/Data"},"timestamp":"2021-07-20T00:55:59.015603Z","kind":"absolute","gauge":{"value":499963174912.0}}
{"name":"filesystem_used_bytes","namespace":"host","tags":{"collector":"filesystem","device":"/dev/disk1s2","filesystem":"apfs","host":"COMP-C02F514NML87","mountpoint":"/System/Volumes/Data"},"timestamp":"2021-07-20T00:55:59.015603Z","kind":"absolute","gauge":{"value":194760007680.0}}
{"name":"filesystem_used_ratio","namespace":"host","tags":{"collector":"filesystem","device":"/dev/disk1s2","filesystem":"apfs","host":"COMP-C02F514NML87","mountpoint":"/System/Volumes/Data"},"timestamp":"2021-07-20T00:55:59.015603Z","kind":"absolute","gauge":{"value":0.40910688042640686}}
```
I get `.389` for "used / total" here, which is only off by `0.11` 🤷

Reply:
0.409 - 0.389 is only 0.020, which is possibly just accounting for reserved space in the ratio.

Reply:
Oof, I can't math good. 👍